utils/src/test/scala/com/salesforce/op/utils/spark/RichMetadataTest.scala

```diff
@@ -100,6 +105,17 @@ class RichMetadataTest extends FlatSpec with TestCommon {
     summaryMeta.containsSummaryMetadata shouldBe true
   }
 
+  it should "create summary for a given metadata" in {
+    val richMetaData = RichMetadata(meta1)
```

Comment:
No need to explicitely wrap metadata into RichMetadata. It should world implicitly.